src/frameworks/react-i18next.ts

```diff
@@ -63,39 +63,11 @@ class ReactI18nextFramework extends Framework {
     '{key}_other',
   ]
 
-  detectHardStrings(doc: TextDocument) {
```

Comment:
@LittleBadBad are you removing the `detectHardStrings` on purpose?

Reply:
sorry, `detectHardStrings` is already recovered, check the [code](https://github.com/LittleBadBad/i18n-ally/blob/feat/args-pre-suf/src/frameworks/react-i18next.ts)